.ci/validate-examples-regression-sample.yml

```diff
@@ -0,0 +1,44 @@
+pool:
```

Comment:
Could we have only one pipeline but with conditions?

Reply:
i think its clearer to have 2 seperate pipelines

Reply:
then you can look at the history easier

Reply:
The problem is that you will need to create two or more Azure Dev Ops pipelines. It would be good if we keep our CI configuration in source code as much as possible.